### PR TITLE
updated sln reference, readme

### PR DESCRIPTION
**Pull request by RicoSaupe**
first small PR. :) 

in regards to issue #580 i discovered also the following. 

1. webpack config was still in default sln file
2. travis.yml still referenced net 6
3. updated readme.md files to reflect used net8 and updated npm, node versions

### CODE REVIEW

**Comment by RicoSaupe** [comment]
Another thought. Do we need to reference these file as solution files? They are generated anyways and not to be edited or is someone changing them? 

package-lock.json
paket.lock

**Comment by isaacabraham** [comment]
@martinbryant these changes look good, especially the `readme.md` which is critical to get correct - it's the docs file that everyone looks at the first time.

File: Content/default/README.md [hunk 1]
```diff
@@ -17,6 +18,15 @@ Before you run the project **for the first time only** you must install dotnet "
 dotnet tool restore
 ```
 
+To check for updates, use
```
Comment (isaacabraham) [inline Content/default/README.md:21]: I'm not sure that this additional section is required - it's just standard `dotnet tool` information.

File: Content/minimal/README.md [hunk 2]
```diff
@@ -26,6 +27,15 @@ dotnet tool restore
 dotnet fable watch src/Client --run npm run start
 ```
 
+To check for updates, use
```
Comment (isaacabraham) [inline Content/minimal/README.md:30]: Ditto above comment.

File: .travis.yml [hunk 3]
```diff
@@ -1,5 +1,5 @@
 language: csharp
-dotnet: 6.0.101
+dotnet: 8.0.100
```
Comment (isaacabraham) [inline .travis.yml:2]: Definitely want this.

File: Content/minimal/.gitignore [hunk 4]
```diff
@@ -13,5 +13,6 @@ node_modules/
 release.cmd
 release.sh
 *.orig
+*.fsproj.user
 *.DotSettings.user
-*.fs.js
\ No newline at end of file
+*.fs.js
```
Comment (isaacabraham) [inline Content/minimal/.gitignore:18]: We probably want `fs.js` in both minimal and full.

File: Content/default/SAFE.App.sln [hunk 5]
```diff
@@ -9,7 +9,6 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution
 		paket.lock = paket.lock
 		package-lock.json = package-lock.json
 		package.json = package.json
-		webpack.config.js = webpack.config.js
```
Comment (isaacabraham) [inline Content/default/SAFE.App.sln:12]: Yes, remove this - not required (in fact, it's wrong to have it there)..

**Review by isaacabraham — CHANGES_REQUESTED** [review]
I think that this is a good set of changes, especially the readme.

File: Content/default/README.md (hunk 1 above)
Comment (RicoSaupe) [reply Content/default/README.md:21]: correct, is just what you should know anyways. I felt it was kind of helpful for me lately. But you are right.

File: Content/minimal/.gitignore (hunk 4 above)
Comment (RicoSaupe) [reply Content/minimal/.gitignore:18]: agree, not sure why this was removed actually

File: Content/default/README.md (hunk 1 above)
Comment (RicoSaupe) [reply Content/default/README.md]: removed this section

File: Content/minimal/README.md (hunk 2 above)
Comment (RicoSaupe) [reply Content/minimal/README.md]: removed this section

**Comment by RicoSaupe** [comment]
@isaacabraham was aligning the project file names to start upper case but if failed here "tests/tests.fsproj". I could rename the file and also the references to it but git ignored the upper lower case file name and did not push the file name change. So you build failed, since its using ubuntu. 
Any idea how to push also the file name changes? If you think its ok.

**Comment by martinbryant** [comment]
@RicoSaupe thanks vert much for the PR. 

I have pushed some changes that fixed the file renaming. Its a bit silly that it doesn't recognise the case change so I committed a change to rename completely and then another to make it correct ;)

**Review by isaacabraham — APPROVED** [review]

**Comment by isaacabraham** [comment]
Thank you both very much!

**Pull request merged**